File: 1_Year/CursoJava/crescente_decrescente.java
import java.util.Scanner;

public class crescente_decrescente {
    public static void main(String[] args) {
        int x=0, y=1;
        Scanner scanf = new Scanner(System.in);
        while (x!=y)
        {
        x = scanf.nextInt();
        y = scanf.nextInt();

        if (x==y) 
            x=y;
        else if (x>y)
            System.out.printf("Decrescente\n");
        else
            System.out.printf("Crescente\n");
        }
        scanf.close();
    }
}

File: 1_Year/AED l/www.beecrowd.com.br/Randon/crescente_decrescente.c
#include <stdio.h>

int main () {
    int x, y;
    while (x!=y)
    {
    scanf("%d %d", &x, &y);
    if (x==y)
        return 0;
    else if (x>y)
        printf("Decrescente\n");
    else
        printf("Crescente\n");
    }
}